W-13015250 remediate instance validation issues bsj (#63)
mulesoft/docs-api-governance-documentation: modules/ROOT/pages/find-and-fix-conformance-issues.adoc

= Finding and Fixing Conformance Issues

> To find conformance issues, use API Governance profiles or CLI commands to validate the APIs against the set of rulesets to which you want your APIs to conform. After you have validated the APIs, you can see conformance badges in API Governance, Exchange, and API Manager and can view a detailed governance report in API Governance and API Manager.

Reviewer: ```suggestion
To find conformance issues, use API Governance profiles or CLI commands to validate the APIs against the set of rulesets to which you want your APIs to conform. After you validate the APIs, you can see conformance badges in API Governance, Exchange, and API Manager and can view a detailed governance report in API Governance and API Manager.
```

> You can click links in the API Governance console or conformance reports to go to the best place to start fixing the conformance issues. For example, to fix specifications, you can follow links to Design Center and to fix instances you can follow links to API Manager.

Reviewer: ```suggestion
You can click links in the API Governance console or conformance reports to find the best place to fix the conformance issues. For example, to fix specifications, follow links to Design Center, and to fix instances, follow links to API Manager.
```

> For details on fixing conformance, refer to:

Reviewer: ```suggestion
For details about fixing conformance:
```
